packages/backend/discovery/apex/config.jsonc

```diff
@@ -50,7 +50,7 @@
       }
     },
     "SHARPVerifierProxy": {
-      "ignoreDiscovery": true
+      "sharedModule": "l2beat-starkware"
```

Comment:
or maybe add a field "sharedModules" to the config, which will be the list of the contract identifiers?

Reply:
Do you mean config.ts?